miner/block_builder.go

```diff
@@ -369,27 +363,30 @@ func (t *BlockBuilder) acceptBlockData() {
 		case <-t.stopChan:
 			return
 
-		case id := <-t.beginRoundEvent:
-			atxID, proofs, err := t.blockOracle.BlockEligible(id)
+		case layerID := <-t.beginRoundEvent:
+			atxID, proofs, err := t.blockOracle.BlockEligible(layerID)
 			if err != nil {
-				t.With().Error("failed to check for block eligibility", log.LayerId(uint64(id)), log.Err(err))
+				t.With().Error("failed to check for block eligibility", log.LayerId(uint64(layerID)), log.Err(err))
 				continue
 			}
 			if len(proofs) == 0 {
-				t.With().Info("Notice: not eligible for blocks in layer", log.LayerId(uint64(id)))
+				t.With().Info("Notice: not eligible for blocks in layer", log.LayerId(uint64(layerID)))
 				continue
 			}
 			// TODO: include multiple proofs in each block and weigh blocks where applicable
 
-			txList := t.TransactionPool.PopItems(MaxTransactionsPerBlock)
-
 			var atxList []types.AtxId
-			for _, atx := range t.AtxPool.PopItems(MaxTransactionsPerBlock) {
+			for _, atx := range t.AtxPool.GetAllItems() {
```

Comment:
why all items? why is this pr changing flows unrelated to TX flow? and even so, please  leave a comment when changing logic

Reply:
Didn't change the behavior, only the name of the function, as the previous name was clearly misleading. I'll ensure we have an issue to add this functionality.